tests/plugins/inlinetoolbar/manual/fallbackstyles.html

```diff
@@ -0,0 +1,104 @@
+<style>
+	body {
+		/* (#1048) */
+		margin-left: 0px;
+		padding-left: 400px;
+	}
+</style>
+<textarea id="editor1">
+	<p>Hello world</p>
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+</textarea>
+<div contenteditable="true" id="editor2">
+	<p>Hello world</p>
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+</div>
+<div contenteditable="true" id="editor3">
+	<p>Hello world</p>
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<img src="%BASE_PATH%_assets/lena.jpg">
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+</div>
+
+<script>
+	function instanceReadyListener( evt ) {
+		var images = this.editable().find( 'img' ).toArray(),
+			panel = new CKEDITOR.ui.inlineToolbar( this );
+
+		panel.addItems( {
+			bold: new CKEDITOR.ui.button( {
+				label: 'test',
+				command: 'bold'
+			} ),
+			language: evt.editor.ui.create( 'Language' ),
+			Styles: evt.editor.ui.create( 'Styles' )
+		} );
+
+		CKEDITOR.tools.array.forEach( images, function( image ) {
+			image.on( 'click', function() {
+				panel.attach( this );
+			} );
+		} );
+	}
+
+	var appendStyleSheet = sinon.stub( CKEDITOR.dom.document.prototype, 'appendStyleSheet', function( cssFileUrl ) {
+		// Simualate missing css in skin.
+		if ( cssFileUrl.indexOf( 'inlinetoolbar.css' ) > -1 || cssFileUrl.indexOf( 'balloonpanel.css' ) > -1 ) {
+			cssFileUrl = '';
+		}
+
+		if ( this.$.createStyleSheet ) {
```

Comment:
We didn't you use call to original function here same as in unit test?

Reply:
I've made it earlier with stub and forget about it. I think option with calling origin function is simpler, and can be resistant for possible method changes.
I've update this to looks like in Unit test.